README.md

```diff
@@ -107,25 +103,37 @@ This module contains all the classes relating to utility for and verification of
 ### library
 This module contains an assortment of Java classes that have been copied over from DDF to support operations that shouldn't be handled by the test code; i.e. signature validation using x509 certificates.
 
-### plugins
-This module contains the API and provider-specific plugin implementations
-needed to interact with IdPs/SPs.
+### external
+This module contains the API for a plugin that needs to be implemented for each external SAML product before these tests
+can be run against it. There are also a few that are already implemented. The generated jar file from these modules
+needs to be installed to a deployment directory of the user's choosing and then referred to by system property when
+running tests.
 
-#### ddf-plugins
-This module contains the ServiceProvider plugins that are used to connect with
-a DDF IdP. It should also be used as the model for building plugins for connecting
-with other IdPs for compliance testing. The generated jar file from this module
-needs to be installed to a deployment directory of the user's choosing and then
-referred to by system property when running tests.
-
-e.g. If the ServiceProvider plugin jar(s) are copied to `/home/saml-conform/deploy`
+e.g. If the ServiceProvider plugin jar is copied to `/home/saml-conform/deploy`
 then the tests should be invoked with `-Dsaml.plugin.deployDir=/home/saml-conform/deploy`.
```

Comment:
❓ I haven't gotten deeper in the review yet to know for sure, but is this necessary any longer? Is the command generating multiple system properties for the runtime still, or have they been collapsed as well as the command line param?

It looks like the two system properties needed for running the tests/suites directly (from the command line or within Intellij) are
`-Dimplementation.path` and `-Dtest.sp.metadata`

Reply:
I fixed up this section. Reworded to be more clear that they need to copy the plugin jar and necessary metadata into a directory then point to that directory by running the test script with `-i <directory>`.